docs/index.md

```diff
@@ -197,16 +180,16 @@ tag:
 
 </details>
 
-:::info Note
-Please note that the definitions above are not complete and are used as a demonstration for you to grasp the basic idea of the multi-file approach.
-:::
+### Validation that meets you where you are
 
-The problem with the multi-file approach is that many existing tools offer multi-file support as the only feature, meaning that you will have yet another tool to install and maintain. OpenAPI CLI has a strong advantage here over other tools, as it bundles files automatically and it's just one of the powerful features it provides you with.
+Linting your OpenAPI definition ensures that it's valid and adheres to a set of rules and standards. This is especially important if you follow the design-first approach to developing APIs. You can use built-in linting rules, create your own custom rules — or a combination of both. Custom rules also extend basic functionality so you can respond to specific use-cases.
 
-### Linting and extensibility
+## Relationship to Redoc
+If OpenAPI CLI's job is to keep you sane as you work with your API definitions, [Redoc's](./docs/quickstart.md) role is to swiftly deploy your docs to a website so all that hard work can be appreciated. There are [multiple deployment options for Redoc](./docs/deployment/intro.md).
 
-Linting is associated with _extensibility_. Linting is used to ensure that your OpenAPI definition is clear and doesn't contain errors or quality issues. To instruct OpenAPI CLI how to detect those issues, you use either built-in or custom rules, or the combination of both. With these rules, you ensure that the OpenAPI documents are consistent, correct, and follow a specific API design standard or style. Furthermore, custom rules can help you extend the basic functionality to cover specific use-cases that your API definitions need to accommodate. Using rule-based linting is especially useful when you follow the design-first API development approach.
+## Read the docs!
+Everything you need to know about Redocly OpenAPI CLI is contained in the following pages. We take as much pride in our docs as we do in our code, so we're always fine-tuning, updating and improving. We suggest you start with our [quickstart guide](./docs/quickstart.md).
 
-## Contributions
+## Want to contribute?
 
-The source code is available in the [OpenAPI-CLI GitHub repository](https://github.com/Redocly/openapi-cli).
+Then join our very active and supportive community! The source code is available [here](https://github.com/Redocly/openapi-cli).
```

Comment:
suggestion: use a more descriptive anchor text instead of "here". Not just for SEO, but also to make it clearer for the readers. For example, if you make it clear in the anchor text that the link leads to a _GitHub_ repository, that tells the reader we manage the project on GitHub, and that they'll need an account there to contribute (and maybe they don't have one yet because they only use GitLab).